docs/src/mdsvex/h2.svelte

```diff
@@ -0,0 +1,23 @@
+<script>
```

Comment:
I hate code duplication. At least the script for h1 - h3 is the same. I recommend moving it to a separate .js file and including it using `<script src="./h.js">` (or similar) instead

Reply:
Aaaaand what exactly do you expect this file to contain?
I feel your pain, at some point I even wanted to set up Rollup to generate those files, but then figured that's even more insane that a little code duplication

Reply:
Exactly the same content:
```js
import { onMount } from 'svelte';

export let id = null;

let href = id && `#${id}`;
onMount(function createSapperCompatiblePermalink() {
  if (id == null) {
    return;
  }
  href = window.location.origin + window.location.pathname + href;
});
```

Reply:
Oh, that's what you mean. Welp, maybe, but it would be a major exception in an established system of file structure, and the benefit of that is quite negligible. Separating the script is especially daunting because the markup uses props that you can't immediately see, which is different from styles, where you don't really use them in the markup or script whatsoever